client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -48,14 +51,20 @@ class PodcastDetails extends Component {
 	}
 
 	renderSaveButton() {
-		const { handleSubmitForm, isRequestingSettings, isSavingSettings, translate } = this.props;
+		const {
+			handleSubmitForm,
+			isRequestingSettings,
+			isSavingSettings,
+			isRequestingCategories,
+			translate,
+		} = this.props;
 		return (
 			<Button
 				compact={ true }
 				onClick={ handleSubmitForm }
 				primary={ true }
 				type="submit"
-				disabled={ isRequestingSettings || isSavingSettings }
+				disabled={ isRequestingSettings || isSavingSettings || isRequestingCategories }
```

Comment:
We will want to be pretty careful with this loading state, especially in later stories when we implement disabled fields and a button to disable podcasting.  Until we have fetched **all** categories from a site, we cannot know whether podcasting is actually enabled or not.